components/login/login.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+import { GoogleAuthProvider, signInWithPopup } from "firebase/auth";
+import { auth } from "../../config/firebaseConfig";
+import { useAuth } from "../../context/authContext";
+
+export const Login = () => {
+  const { user, setUser }: any = useAuth();
+
+  const googleProvider = new GoogleAuthProvider();
+
+  function handleClick() {
+    if (user.uid) {
+      console.log("Already Logged In");
```

Comment:
Remove `console.log()` statements.

Reply:
Yes, Logged a statement, until I develop the notification functionality.